AccessibilityTests/Sources/AccessibilityTests.swift

```diff
@@ -78,7 +84,12 @@ final class AccessibilityTests: XCTestCase {
         }
     }
     
+    private static func isMatrixString(_ string: String) -> Bool {
```

Comment:
Minor, but this seems more descriptive to me.
```suggestion
    private static func isMatrixIdentifier(_ string: String) -> Bool {
```